content/Gemfile.lock

```diff
@@ -15,7 +15,7 @@ GEM
     capybara (2.4.4)
       mime-types (>= 1.16)
       nokogiri (>= 1.3.3)
-      rack (>= 1.0.0)
+      rack (>= 1.6.12)
```

Comment:
Please revert all the gemfile changes -- those are fairly disruptive and require building and shipping a new Docker container, so they're not something that can be done as an aside in an unrelated PR.

Reply:
sounds good, new commit will revert this change. will be happy to pair with you when we do choose to update the gem!